### ISSUE TITLE
update latest version on win7 it's not work

### ISSUE THREAD

**Issue by orange-jacky** [desktop/desktop#7035]
last night, i update desktop on my win7 os, after restart, it's not work. can't load local resp
you can reproduce it on your win7 os

**Comment by tierninho** [comment]
@orange-jacky To get some more information about why the installation is failing it would be helpful if you could send over the log file.

You should be able to find the log file here:

`%LocalAppData%\GitHubDesktop\SquirrelSetup.log`

If no log file is being generated there you should be able to find one here:

`%LocalAppData%\SquirrelTemp\SquirrelSetup.log`

Please upload that file as an attachment.

You are running Windows 7, with 64-bit or 32-bit?

**Comment by Lucifer1000** [comment]
Hi, I have same issue on the latest 1.6.3 version.
desktop app can be installed and executed, but my local clone repository can't be located.
the memory pop-up occurred. here are the log you requested.
[SquirrelSetup.log](https://github.com/desktop/desktop/files/2944568/SquirrelSetup.log)
[SquirrelSetup.log](https://github.com/desktop/desktop/files/2944570/SquirrelSetup.log)

**Comment by orange-jacky** [comment]
[SquirrelSetup.log](https://github.com/desktop/desktop/files/2945772/SquirrelSetup.log)

**Comment by steveward** [comment]
@orange-jacky @Lucifer1000 could you also grab the log file from GitHub Desktop itself? To access the log files go to the file menu in GitHub Desktop and select `Help` > `Show logs in Explorer`. 

The log files are created daily -- please send over a log file as an attachment from a day where you experienced the issue.

Similar reports of issues with Windows 7 installs in https://github.com/desktop/desktop/issues/7038.

**Comment by orange-jacky** [comment]
[2019-02-19.desktop.production.log](https://github.com/desktop/desktop/files/2948477/2019-02-19.desktop.production.log)
[2019-02-20.desktop.production.log](https://github.com/desktop/desktop/files/2948478/2019-02-20.desktop.production.log)
[2019-02-21.desktop.production.log](https://github.com/desktop/desktop/files/2948479/2019-02-21.desktop.production.log)
[2019-02-22.desktop.production.log](https://github.com/desktop/desktop/files/2948480/2019-02-22.desktop.production.log)
[2019-02-23.desktop.production.log](https://github.com/desktop/desktop/files/2948481/2019-02-23.desktop.production.log)
[2019-02-24.desktop.production.log](https://github.com/desktop/desktop/files/2948482/2019-02-24.desktop.production.log)
[2019-02-25.desktop.production.log](https://github.com/desktop/desktop/files/2948484/2019-02-25.desktop.production.log)
[2019-02-26.desktop.production.log](https://github.com/desktop/desktop/files/2948485/2019-02-26.desktop.production.log)
[2019-03-03.desktop.production.log](https://github.com/desktop/desktop/files/2948486/2019-03-03.desktop.production.log)
[2019-03-04.desktop.production.log](https://github.com/desktop/desktop/files/2948487/2019-03-04.desktop.production.log)
[2019-03-05.desktop.production.log](https://github.com/desktop/desktop/files/2948488/2019-03-05.desktop.production.log)
[2019-03-06.desktop.production.log](https://github.com/desktop/desktop/files/2948489/2019-03-06.desktop.production.log)
[2019-03-07.desktop.production.log](https://github.com/desktop/desktop/files/2948490/2019-03-07.desktop.production.log)
[2019-03-09.desktop.production.log](https://github.com/desktop/desktop/files/2948491/2019-03-09.desktop.production.log)

**Comment by shiftkey** [comment]
We have published `1.6.4-beta1` just now which reverts the embedded Git in Desktop from `2.20.1.windows.1` to `2.19.2.windows.1`, which we believe will address this issue.

In the process of investigating we've identified at least two known issues with `2.20.1.windows.1` that have since been fixed in Git for Windows:

 - https://github.com/git-for-windows/git/issues/1989
 - https://github.com/git-for-windows/git/issues/2075

Because of the complexity of the setup to trigger these issues **we are looking for any Windows users to help us confirm this problem has been resolved, which requires installing [the latest beta](https://github.com/desktop/desktop#beta-channel) and testing their problem repositories again**.

We plan to make this available today to everyone, but it would be great to know in advance that these specific issues are addressed.

Thanks again for working with us to get this resolved!
